### PR TITLE
AB#35078: Submission hangfire implementation

### PR DESCRIPTION
**Pull request by blargly**
Ready for review

### CODE REVIEW

**Comment by blargly** [comment]
Still need to sort out the configuration bits which set it to use hangfire.

Also needs ISubmissionService rolling back, as I made many of the methods private not realising they're used elsewhere.  maybe it needs to be split up> Old function logic could go in its own service, though not sure its warranted

File: src/Core/Submissions/Services/SubmissionService.cs [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+
+        // Automapper
+        private readonly IMapper _mapper;
+
+        // Entity Validation and Write Services
+        private readonly IDiagnosisWriteService _diagnoses;
+        private readonly ISampleWriteService _samples;
+        private readonly ITreatmentWriteService _treatments;
+        private readonly IErrorService _errors;
+
```
Comment (beforan) [inline src/Core/Submissions/Services/SubmissionService.cs:34]: lots of constructor injection needed?

**Review by beforan — CHANGES_REQUESTED** [review]
Also need to delete/modify some CI bits:

- labeler configuration
- Submissions AzFunctions build workflow
- Submissions AzFunctions artifact bits inside `main.release.yml`

you should be able to mostly bae these changes on the equivalent ones in #234 or #235 or #228

**Review by Flemingjp — COMMENTED** [review]
Will approve once outstanding comments are resolved

**Review by Flemingjp — APPROVED** [review]

File: .github/labeler.yml [hunk 2]
```diff
@@ -11,6 +11,9 @@
   - src/Core/Jobs/AnalyticsJob.cs
 "Area: Submissions":
   - src/Submissions/**/*
+  - src/Core/Jobs/RejectJob.cs
```
Comment (beforan) [inline .github/labeler.yml:14]: what about staging?

**Review by beforan — CHANGES_REQUESTED** [review]

**Review by beforan — APPROVED** [review]

**Review by Flemingjp — APPROVED** [review]

**Pull request merged**